package.json

```diff
@@ -25,12 +25,13 @@
   "license": "MIT",
   "dependencies": {
     "axios": "^0.18.0",
-    "bitcoinjs-lib": "^3.3.2",
     "blockcypher": "ChluNetwork/node-client",
+    "brorand": "^1.1.0",
     "chlu-did": "ChluNetwork/chlu-did#remove-dead-code",
     "chlu-wallet-support-js": "ChluNetwork/chlu-wallet-support-js",
     "commander": "^2.15.1",
     "debounce-async": "^0.0.1",
+    "elliptic": "^6.4.0",
```

Comment:
Here we are changing our crypto from secp256k1 to ed2559. I am not sure we should make this switch anymore.

Reply:
Elliptic supports secp256k1 as well. It's also what's used in orbit-db.

However, in chlu-did we use ed2559 here: https://github.com/ChluNetwork/chlu-did/blob/master/src/index.js#L8

Let me know if you think secp256k1 is better and if you want us to use it in chlu-did too.